src/test/java/org/killbill/billing/plugin/adyen/TestRemoteBase.java

```diff
@@ -89,13 +89,14 @@
     protected Properties properties;
     protected String merchantAccount;
 
-    @BeforeClass(groups = {"slow", "integration"})
-    public void setUpBeforeClassCommon() throws Exception {
+    @BeforeClass(groups = {"slow"})
```

Comment:
It would be easier for the merge into `master` to take the latest fix instead, see https://github.com/killbill/killbill-adyen-plugin/commit/77f9a6811726069edf24c85aedbd28b95154d850#diff-9297c9fce44c5cf6a43ccaf9fae29ebc.

Reply:
merged that commit to the PR branch.